package.json

```diff
@@ -4,6 +4,8 @@
     "valid_gtin": "./dist/cli.js"
   },
   "scripts": {
+    "clean": "rm -rf dist",
```

Comment:
This will propably not work on Windows machines. Maybe consider to use a dependency called [`rimraf`](https://www.npmjs.com/package/rimraf)